rootfs/etc/deis-dockerbuilder.yaml

```diff
@@ -11,14 +11,18 @@ spec:
   containers:
     - name: repo_name
       imagePullPolicy: Always
-      image: smothiki/dockerbuilder:5.5
+      image: arschles/dockerbuilder:5.5
       env:
         - name: DEBUG
           value: "1"
         - name: TAR_URL
           value: tar-url
-        - name: IMAGE
+        - name: IMG_NAME
           value: imagename
+        - name: ACCESS_KEY_FILE
+          value: /var/run/secrets/object/store/access_key
+        - name: ACCESS_SECRET_FILE
+          value: /var/run/secrets/object/store/access_secret
```

Comment:
passing the location of the volumes to code to add some flexibility. [boot.go in dockerbuilder](https://github.com/deis/dockerbuilder/pull/1/files#diff-f6ee15f775b88714fe6122ff877b9e42R46) reads those env vars and `cat`s the files like the shell script used to. in theory an operator could put the volumes anywhere they want or provision the machines with the key and secret outside of k8s